src/components/Calculator.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
[OPTIONAL]

- [ ] Instead of having this many disabled eslint plus the ones in other modules, it'll be nice to try fix the errors, and in so doing, learning valuable new concepts.